frontend/javascripts/oxalis/model/sagas/volumetracing_saga.js

```diff
@@ -55,10 +55,13 @@ function* warnOfTooLowOpacity(): Saga<void> {
   if (yield* select(state => state.tracing.volume == null)) {
     return;
   }
-
-  const isOpacityTooLow = yield* select(
-    state => state.datasetConfiguration.segmentationOpacity < 10,
-  );
+  const segmentationLayerName = yield* call([Model, Model.getSegmentationLayerName]);
```

Comment:
I think, you can simply `return` if `segmentationLayerName` is null. This should never happen anyway (since we already know there is a volume tracing when we hit this code path).